tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb

```diff
@@ -757,19 +677,7 @@
     "# widen how much text is presented of each tweet\n",
     "pd.options.display.max_colwidth = 250\n",
     "samples = df_tmp.sample(100)\n",
-    "samples\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "f7024cc3",
-   "metadata": {
-    "execution": {}
-   },
-   "outputs": [],
-   "source": [
-    "df = df_tmp\n"
```

Comment:
I think this is needed to use df_tmp else df is overwritten in the exercice cells / student bit so the df / df_tmp mechanism needs to stay